### PR TITLE
Fix release tarball compression

### PR DESCRIPTION
**Pull request by Markos-Th09**
The release tarballs are not actually gzip compressed. This has pretty much been a bug in the workflow since forever.

### CODE REVIEW

File: core/src/scanner.rs [hunk 1]
```diff
@@ -564,9 +564,7 @@ const SYMBOLS: SymbolsMap = generate_map(&[
 	(
 		':',
 		SymbolType::Symbols(
-			generate_map(&[
```
Comment (Maiori44) [inline core/src/scanner.rs:567]: why is this changed?

**Review by Rosy-iso — CHANGES_REQUESTED** [review]
Just found a little odd thing

File: core/src/scanner.rs (hunk 1 above)
Comment (Markos-Th09) [reply core/src/scanner.rs:567]: maybe it got the changes from fixing the duplication in the TokenType enum, and rustfmt formated this

**Review by Rosy-iso — APPROVED** [review]
Everything else seems okay
